drivers/staging/media/rpivid/rpivid.c

```diff
@@ -38,12 +38,14 @@ static const struct rpivid_control rpivid_ctrls[] = {
 	{
 		.cfg = {
 			.id	= V4L2_CID_MPEG_VIDEO_HEVC_SPS,
+			.ops = &rpivid_hevc_sps_ctrl_ops,
```

Comment:
Indentation? Or is it Github messing about?

Reply:
Indented